uml/state_diagrams/power_management.plantuml

```diff
@@ -0,0 +1,41 @@
+@startuml
+
+skinparam shadowing false
+
+title Power Management State Diagram
+
+state "Power Modes" as Pm {
+    state "Full Power" as Fp : entry: Set Display Brightness to Stored State\nentry: Set Bluetooth to Stored State\nentry: Set Health Notification to Stored State\ndo: Display Activated Display View
+    state "Low Power" as Lp : entry: Trigger Motor Vibration for 2 secs\nentry: Set Display Brightness State = Low\nentry: Disable Bluetooth\nentry: Disable Health Notification\ndo: Turn off display
+
+    Fp -down-> Lp : [Battery SoC <= 10% ||\n Accelerometer Force < XN (5 secs)]
+    Lp -up-> Fp : [Battery SoC > 10% &&\nAccelerometer Force >= XN]
+    
+    state Choice2 <<choice>>
+    [*] --> Choice2
+    Choice2 -left-> Fp : [Battery SoC > 10%]
+    Choice2 -right-> Lp : [Battery SoC <= 10% &&\nBattery SoC > 2%]
+
+}
+
+state "Power Down" as Pd : entry: Turn Off Device
+
+state "Battery Error" as Bu : entry: do: Display warning message
+
+state Choice <<choice>>
+[*] --> Choice : Hold Button A\nfor 3 secs
+Choice -down-> Bu : [Battery\nError]
+Choice -left-> Pd : [Battery Soc <= 2%]
+Choice -down-> Pm : [Battery Soc > 2%]
+
+Bu -down-> Pm : [Battery SoC >= 2%]
+Pm -up-> Bu : [Battery\nError]
+
+Bu -down-> Pd : [Battery SoC < 2%]
+
+
+Pm -down-> Pd : [Battery SoC <= 2%]
```

Comment:
I probably also need to account for manual deactivation of device by the user, as shown in the Display state diagram